packages/pictograms/categories.yml

```diff
@@ -1,531 +1,531 @@
 categories:
   - name: Action
     members:
-    - cloud--download
-    - cloud--upload
-    - crop
-    - delete
-    - do--not
-    - download
-    - download--alt
-    - embed
-    - export
-    - export--alt
-    - heart
-    - launch
-    - maximize
-    - minimize
-    - paper--clip
-    - refresh
-    - renew
-    - repeat
-    - reset
-    - trash
-    - upload
-    - upload--alt
+      - cloud--download
```

Comment:
Do you have prettier enabled for YML files by any chance?

Reply:
I thought we did since these changes were formatted on save. I only actually changed line 48 and then the other spaces were added automatically.

Reply:
@jnm2377 I don't think we have it enabled yet for YML files https://github.com/carbon-design-system/carbon/blob/master/package.json#L21